src/mintpy/prep_nisar.py

```diff
@@ -61,9 +61,9 @@ def load_nisar(inps):
     print(f"Found {len(input_files)} unwrapped files")
 
     if inps.subset_lat:
-        bbox = (inps.subset_lat[0], inps.subset_lon[0], inps.subset_lat[1], inps.subset_lon[1])
+        bbox = (inps.subset_lon[0], inps.subset_lat[0], inps.subset_lon[1], inps.subset_lat[1])
```

Comment:
so this isn't converting from SNWE to (left, bottom, right top), it's just fixing the ordering for (left, bottom, right, top)?

Reply:
yes the input arguments follows mintpy subsetting with two arguments `--sub-lat` and `--sub-lon `. They are ordered to make bbox here